.github/workflows/sync-version-with-api-docs.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will open a PR in https://github.com/instill-ai/protobufs updating the API version in the OpenAPI specifications.
+
+name: Sync release version with OpenAPI docs
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  update-api-docs:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout protobufs repository
+      uses: actions/checkout@v4
+      with:
+        repository: instill-ai/protobufs
+        token: ${{ secrets.botGitHubToken }}
```

Comment:
@Sarthak-instill checking other actions, I assume this token exists in this repo and has permissions to fetch and push from / to other Instill AI repos. Am I right?

Reply:
@jvallesm Yes, This token is in working condition, you can use it.

Reply:
Great! Can I get an approval then? I don't have permissions to merge the branch without at least one ✅ Thanks! 🙇